reactnative/README.md

```diff
@@ -0,0 +1,11 @@
+# Hyperbeam React Native example
+
+This is a simple example of how to use Hyperbeam in a React Native app.
```

Comment:
can you please a brief explanation about:
1. debugging locally
2. an explanation why expo is a bad idea (you cannot debug locally with it on android)
3. any other "gotchas" when doing this

Reply:
Done